### PR TITLE
Add Uzbekistan map

### PR DESCRIPTION
**Pull request by Disctanger**
Thank you for creating and maintaining SVG maps. I have tried to add map of Uzbekistan. Please take a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ A set of SVG maps that can be installed via [npm](https://www.npmjs.com) or [yar
 * 🇺🇸 [USA](packages/usa)
 * 🇺🇸 [USA, Counties](packages/usa.counties)
 * 🇺🇸 [USA, Utah](packages/usa.utah)
+* 🇺🇿 [Uzbekistan, Regions](packages/uzbekistan)
```
Comment (VictorCazanave) [inline README.md:45]: Are there smaller or larger administrative divisions (e.g. counties, departments...)?
If yes, could you rename the package `[Uzbekistan, Regions](packages/uzbekistan.regions)` please?
If no, could you change the displayed name `[Uzbekistan](packages/uzbekistan)` please?

File: packages/uzbekistan/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* Sirdaryo
+* Surxondaryo
+* Toshkent
+* Xorazm
+
+### Sea area:
+* Orol Dengizi
+
+### Autonomous Republic:
+* Qoraqalpog‘iston
```
Comment (VictorCazanave) [inline packages/uzbekistan/README.md:25]: Could you use [English names](https://en.wikipedia.org/wiki/Regions_of_Uzbekistan) (in the SVG too) please?

File: packages/uzbekistan/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### yarn
+
+`yarn add @svg-maps/uzbekistan`
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/japan) under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
```
Comment (VictorCazanave) [inline packages/uzbekistan/README.md:41]: Could you indicate the real credits please?

File: packages/uzbekistan/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### Changes
+
+* Remove unnecessary attributes
+* Replace title by name attributes
+* Add viewBox
+* Rename ids
+* Sort `<path/>` alphabetically
```
Comment (VictorCazanave) [inline packages/uzbekistan/README.md:49]: Could you indicate the real changes please?

File: packages/uzbekistan/uzbekistan.svg [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+		name="Andijon"
+		id="andijan"
```
Comment (VictorCazanave) [inline packages/uzbekistan/uzbekistan.svg:9]: Could you use the same name for `name` and `id` (same for all the `<path>`) please?

**Review by VictorCazanave — COMMENTED** [review]
Thanks a lot for your contribution @Disctanger ! 😃

Just some things to fix, then I'll glad to merge it.

**Comment by Disctanger** [comment]
@VictorCazanave Thank you for reviewing the code. I have addressed all of your comments. Please take a look.

File: README.md (hunk 1 above)
Comment (Disctanger) [reply README.md]: Basically, there are no smaller or larger administrative divisions. There is one autonomous republic in Uzbekistan but is it always come together with regions. Therefore i did not separate them.
I have i have removed ", Regions" part of the code and kept the package name as it is.

File: packages/uzbekistan/README.md (hunk 2 above)
Comment (Disctanger) [reply packages/uzbekistan/README.md]: Done

File: .all-contributorsrc [hunk 6]
```diff
@@ -165,6 +165,15 @@
...
       ]
+    },
+    {
+      "login": "Disctanger",
+      "name": "Disctanger",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/10447714?s=460&u=88f7c24ff87d9ce9b552e828c09f1af8d43df105&v=4",
+      "profile": "https://github.com/Disctanger",
+      "contributions": [
+        "code"
+      ]
```
[the remark this answers is not in the document]
Comment (Disctanger) [reply .all-contributorsrc]: I was bit confused about this part. I have removed all the changes from this file.

File: packages/uzbekistan/uzbekistan.svg (hunk 5 above)
Comment (Disctanger) [reply packages/uzbekistan/uzbekistan.svg:9]: Done

File: packages/uzbekistan/README.md (hunk 4 above)
Comment (Disctanger) [reply packages/uzbekistan/README.md:49]: Done

File: packages/uzbekistan/README.md (hunk 3 above)
Comment (Disctanger) [reply packages/uzbekistan/README.md]: Done. I got the SVG from the source, therefore i have updated just the link.

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @Disctanger for code

**Pull request merged**